ffs/scheduler/scheduler.go

```diff
@@ -207,7 +210,31 @@ func (s *Scheduler) run() {
 	}
 }
 
-func (s *Scheduler) scanRenewable(ctx context.Context) {
+func (s *Scheduler) execRepairCron(ctx context.Context) {
```

Comment:
This is the moment that prior work pays off. The repair job is scheduling the last known _CidConfig_ for a repairable _CidConfig_ (has a new _Repairable_ attribute set to `true`).
Since executing _CidConfigs_ is a best-effort work to ensure that configuration is true, that has the same semantics of a repair action... so basically pushing the last known config again will repair things of something got broken.

Reply:
💯